agent/gather.sh

```diff
@@ -6,15 +6,16 @@ SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
 
 source "$SCRIPTDIR"/common.sh
 
-for ip in $(cat "${OCP_DIR}"/hostip); 
+for host in $(cat "${OCP_DIR}"/hostname); 
 do 
-    echo "Trying to gather agent logs on host ${ip}"
-    if ssh -o 'ConnectTimeout=30' -o 'StrictHostKeyChecking=no' -o 'UserKnownHostsFile=/dev/null' core@"${ip}" agent-gather -O >agent-gather-"${ip}".tar.xz; then
-    echo "Agent logs saved to agent-gather-"${ip}".tar.xz" >&2
+    echo "Trying to gather agent logs on host ${host}"
+    if ssh -o 'ConnectTimeout=30' -o 'StrictHostKeyChecking=no' -o 'UserKnownHostsFile=/dev/null' core@"${host}" agent-gather -O >agent-gather-"${host}".tar.xz; then
+        echo "Agent logs saved to agent-gather-"${host}".tar.xz" >&2
     else
         if [ $? == 127 ]; then
-            echo "Skipping gathering agent logs, agent-gather script not present on host ${ip}." >&2
+            echo "Skipping gathering agent logs, agent-gather script not present on host ${host}." >&2
         fi
-        rm agent-gather-"${ip}".tar.xz
+        rm agent-gather-"${host}".tar.xz
     fi
+    sudo sed -i "/${host}/d" /etc/hosts
```

Comment:
Thinking about this some more, we should probably remove this in agent/cleanup.sh since there is no guarantee that gather will be run after these host entries are added.

Reply:
right